src/transformers/modeling_utils.py

```diff
@@ -1691,6 +1715,16 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 If `True`, will temporarily offload the CPU state dict to the hard drive to avoid getting out of CPU
                 RAM if the weight of the CPU state dict + the biggest shard of the checkpoint does not fit. Defaults to
                 `True` when there is some disk offload.
+            load_in_8bit (`bool`, *optional*):
+                If `True`, will convert the loaded model into mixed-8bit quantized model. To use this feature please
+                install `bitsandbytes` compiled with your CUDA version by running `pip install -i
+                https://test.pypi.org/simple/ bitsandbytes-cudaXXX` where XXX is your CUDA version (e.g. 11.6 = 116).
+                Make also sure that you have enough GPU RAM to store half of the model size since the 8bit modules are
+                not compiled and adapted for CPUs.
+            int8_threshold (`int`, *optional*):
```

Comment:
Maybe give more details here on what it actually means so that the users do not have to read the paper to be able to use it.

Reply:
Thank you for pointing out this! 
I added more description on 7abb914e726e4f4d3845d4faed2c0085b5e73e13 and 961e57ea85d943b0316f9f69e43dff6e7f0915a4 that should let user understand the usage of this parameter.

Reply:
If you want to add more information, you could write:
>Values are usually normally distributed, that is, most values are in the range [-3.5, 3.5], but there are some exceptional systematic outliers that are very differently distributed for large models. These outliers are often in the interval [-60, -6] or [6, 60]. Int8 quantization works well for values of magnitude ~5, but beyond that, there is a significant performance penalty. A good default threshold is 6, but a lower threshold might be needed for more unstable models (small models, fine-tuning).

Reply:
Nice thanks! Added some suggested comments here: 0324f4b1885e51cc93f3f25754b1469f7538bbc7